build.gradle

```diff
@@ -16,7 +16,7 @@ plugins {
 
 // TODO(PF-1981) - Move version to settings.gradle
 // The tools/publish-release.sh script depends on this version string being of the format "version = '1.2.3'"
-version = '0.258.0'
```

Comment:
Changes in non-shell files are due to rebase. They belong to 2 previous PRs and will not cause merge issues
since this PR already had comments, retaining it.